Werewolf for Telegram/Shared/Roles.cs

```diff
@@ -141,6 +141,9 @@ public enum IRole : long // NOTE: IN ROLE CONFIGURATION, FALSE WILL MEAN ENABLED
 
         [Role("🎃", false)]
         Spumpkin = 8796093022208,
+
+        [Role("🐐🌚")]
```

Comment:
The 🌚 emoji is already used for the Wolf Man - it's better to avoid duplications like this.
And while the 🐐 emoji makes sense in English, it most likely does not in other languages.
How about this one, for example:
```suggestion
        [Role("😰")]
```